### DISCUSSION TITLE
Acces to my repository

### DISCUSSION THREAD

**Discussion by Suhoy571** [community/community#41983] [Repositories]
### Select Topic Area

Product Feedback

### Body

Hi. I current;y locate in LPR, so my private repositories was blocked. When I want to change visibility of my repo the buttont doest work. So, what I need to do to access my code?
![image](https://user-images.githubusercontent.com/17066007/208241736-c088e83e-6a64-4a26-948c-49808ee033f4.png)

**Comment by jfullstackdev** [comment]
can you have the full screen screenshot ? 

what you should be seeing is like this 

![image](https://user-images.githubusercontent.com/47092464/208246919-f8ea5dd8-7804-49ad-b08f-da6106845f71.png)
